src/models/whisper.cpp

```diff
@@ -42,7 +38,7 @@ Whisper_State::Whisper_State(const Whisper_Model& model, DeviceSpan<int32_t> seq
   }
 
   if (inputs.alignment_heads != nullptr) {
-#if USE_CUDA
+#if 0  // USE_CUDA
```

Comment:
Will these USE_CUDA's be removed?

Reply:
Yep, that will happen when Kunal merges his whisper branch